scribe-tool/scrawl.js

```diff
@@ -368,6 +371,47 @@
     scrawl.htmlFooter = footer;
   };
 
+  scrawl.preprocessLine = function(context, lines, lineI)
+  {
+    const line = lines[lineI];
+    const match = commentRx.exec(line);
+    if(!match)
+    {
+      return;
+    }
```

Comment:
We really need to re-apply a modern code formatting to all of this code. Just making a comment, no need to do it now.